packages/core/src/shared/globalState.ts

```diff
@@ -31,6 +31,8 @@ type globalKey =
     | 'aws.amazonq.codewhisperer.newCustomizations'
     | 'aws.amazonq.hasShownWalkthrough'
     | 'aws.amazonq.showTryChatCodeLens'
+    | 'aws.amazonq.notifications'
+    | 'aws.toolkit.notifications'
```

Comment:
since each extension has its own globalState, do we actually need 2 keys or would one `aws.notifications` key work? that would also simplify a fair amount of code AFAICT.